mw-webapp/src/dataAccessLogic/HealthCheckDAL.ts

```diff
@@ -0,0 +1,33 @@
+import {HealthCheckService} from "src/service/HealthCheckService ";
+
+/**
+ * Interface for API health check status
+ */
+export interface HealthCheckStatus {
+
+  /**
+   * Flag indicating whether the API is working
+   */
+  isWorkingApi: boolean;
+}
+
+/**
+ * HealthCheckDAL
+ */
+export class HealthCheckDAL {
+
+  /**
+   * Method to check the health of the API
+   */
+  public static async checkApiHealth(): Promise<HealthCheckStatus> {
+    try {
+      const healthCheckDTO = await HealthCheckService.getHealthCheck();
+      const isWorkingApi = healthCheckDTO.message === "The way APi is working fine";
+
+      return {isWorkingApi};
+    } catch (error) {
+      return {isWorkingApi: false};
+    }
```

Comment:
you can delete it because line 25 return always boolean value

Reply:
fixed